css/nav.css

```diff
@@ -0,0 +1,15 @@
+nav>.nav-icon>a {
+    letter-spacing: 0.2rem;
+}
+
+nav>ul {
+    margin-left: auto;
+}
+
+nav>input {
+    flex-grow: 1;
+}
+
+nav {
+    height: 4rem;
+}
```

Comment:
Suggestion: better to use class selectors

Reply:
This is intentional because the nav is uniform through the site and that is the reason I have used child combinators to target specific elements.